frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -105,26 +105,26 @@ const PlatformWrapper = ({
 # Enrollment
 --host_identifier=instance
 --enroll_secret_path=secret.txt
---enroll_tls_endpoint=/api/latest/osquery/enroll
+--enroll_tls_endpoint=/api/osquery/enroll
```

Comment:
@lukeheath 

Something to double check:

IIRC these had the `v1` in the path, from the history: https://github.com/fleetdm/fleet/commit/a8d7b5478b7620b0974236666d25b55c25f36a8c

Here's Fleet server registering the endpoints:
https://github.com/fleetdm/fleet/blob/1f5413ed1762eed0f483a293483c589cfd600372/server/service/handler.go#L402-L418

Reply:
chiming in to give a heads-up: Tomas mentioned in Slack that either `/v1/osquery/enroll` or `/osquery/enroll` were are, but if we need the `v1` then there are multiple places in the code using `/osquery/enroll` that we should update

Reply:
on a second read, he mentioned that both URLs exist, not that both are the same. Is there a difference?

Reply:
@lucasmrod Looking at `handler.go` it appears that all `/api/osquery/` endpoints support no version or a `v1` version. We are going to retire `v1` versioning at Fleet 5, so it's preferable to use the non-versioned endpoints here so we don't have to update it later.

If the endpoints are not all available without a version, please let me know as we'd need to update the flagfile.

Reply:
+1, disregard my warning :)